Sources/RxAlamofire/RxAlamofire.swift

```diff
@@ -713,7 +713,9 @@ public extension Reactive where Base: Alamofire.Session {
           if let error = response.error {
             observer.on(.error(error))
           } else {
-            observer.on(.completed)
+            DispatchQueue.global(qos: .background).async {
```

Comment:
Oh, I just remembered a GCD issue I saw somewhere - if we call `DispatchQueue.global` frequently enough, this can spawn an uncontrollably large number of additional threads. let's just create a static queue reference to a concurrent queue